src/braket/pennylane_plugin/translation.py

```diff
@@ -57,19 +40,177 @@ def supported_operations() -> FrozenSet[str]:
     Returns:
         FrozenSet[str]: The names of the supported operations
     """
-    return frozenset(_OPERATION_MAP.keys())
+    return frozenset(op.__name__ for op in translate_operation.registry)
```

Comment:
I think `translate_operation.registry` also contains `object`, which shouldn't be there.